.env.d.ts

```diff
@@ -0,0 +1,10 @@
+// Declare environment variable types
+
+namespace NodeJS {
+  interface ProcessEnv {
+    RECORDS_URL: string;
+    NODE_ENV: 'development' | 'production';
```

Comment:
nextjs.org/docs/messages/non-standard-node-env makes it look to me like Next more or less manages NODE_ENV for us to some reasonable degree, so it may not make sense to encourage tweaking it by hand until we get to the point where think there's a good reason to do that. Was there some particular reason you included it? If not, maybe just remove it...